runtime/include/j9cfg.h.ftl

```diff
@@ -42,7 +43,12 @@ extern "C" {
 #undef EsVersionMinor
 #define EsVersionMinor ${uma.buildinfo.version.minor}0
 
-#define EsVersionString "${uma.buildinfo.version.major}.${uma.buildinfo.version.minor}"
+#ifdef VENDOR_VERSION_STRING
+#define EsVersionString VENDOR_VERSION_STRING
+#else
+#define EsVersionString "0.8.0-internal"
```

Comment:
I don't think we want to hardcode this value or we'll need to continual update it for each release and each milestone.  Is there a way we can generate it based on the branch name / sha / tag?

ie: For the 0.8 release, we'll create a `openj9-0.8` branch, and then apply tags like `openj9-0.8RC1` to that branch.  It would be nice if this version string used those tags for the release branches and maybe `openj9-master-sha` (?) for the master branch.

Reply:
Agreed. Working a way to grab `openj9` branch/tag/sha info before invoking target `j9vm-build`, and pass such info `$(VENDOR_VERSION_STRING)` to the target via `VENDOR_VERSION_STRING=$(VENDOR_VERSION_STRING)`.

Reply:
Updated PR to use `OpenJ9 tag` or `branch/sha` when `tag` is not available.
Pls review again.